cli/oyster-cvm/src/commands/simulate.rs

```diff
@@ -0,0 +1,676 @@
+use alloy::primitives::Address;
+use alloy::providers::ProviderBuilder;
+use alloy::signers::k256::sha2::{Digest, Sha256};
+use anyhow::{anyhow, Context, Result};
+use clap::Args;
+use reqwest::Client;
+use serde::Deserialize;
+use serde_yaml::Value;
+use std::collections::HashSet;
+use std::fs::{self, File};
+use std::io::{BufRead, BufReader, Write};
+use std::path::{Component, PathBuf};
+use std::process::{Command, Stdio};
+use std::str::FromStr;
+use std::sync::{Arc, Mutex};
+use std::thread;
+use std::time::Duration;
+use tracing::info;
+
+use crate::configs::global::{ARBITRUM_ONE_RPC_URL, OYSTER_MARKET_ADDRESS};
+use crate::types::Platform;
+use crate::utils::market::{Operator, OysterMarket};
+
+const DOCKER_IMAGE_CACHE_PATH: &str = "docker_images_cache";
```

Comment:
docker_images_cache location should be different because if it is in local folder then it will be added as context to the docker image as well which will unnecessarily increase the size of docker image.